src/constants.js

```diff
@@ -2,7 +2,7 @@
 
 declare var __DEV__: ?string
 
-export const SC_ATTR = 'data-styled-components'
+export const SC_ATTR = process.env.SC_ATTR || 'data-styled-components'
```

Comment:
I think this might lead to minor issues in some setups where not all `process.env` variables are replaced; specifically setups where only `NODE_ENV` is set up.

@mxstbr We might want to either test this against such a setup or just use `typeof process !== 'undefined'` for now. Agreed?

Reply:
@kitten rollup already defines `process.env.NODE_ENV`, so `process.env` can't be undefined.

Reply:
We’re using the replace plugin there. So it’ll just search for it as a string and replace it. The same won’t go for this new env variable. Thus I’m pretty sure we can’t make the assumption that process will be defined

Reply:
@kitten ok, I see it now.

But, if we will use something like this:
```js
export const SC_ATTR = (process && process.env && process.env.SC_ATTR) || 'data-styled-components'
```

after processing with application bundler this code may look like that:

```js
var SC_ATTR = (process && process.env && 'data-some-custom-styled-components') || 'data-styled-components'
```

And in that case we rely on the fact that `process` defined in runtime, which is not guaranteed with bundlers like rollup. Now I think, that custom `process.env` is not good for browser libraries.

Another question: Is it necessary to have `data-` within `SC_ATTR`?